scripts/build_test.py

```diff
@@ -119,7 +118,7 @@ def test_ensure_directory_exists(self):
         build.ensure_directory_exists(random_filepath)
         # Asserting ../oppia/random exists.
         self.assertTrue(os.path.isdir(SANDBOX_DIR))
-        # Clean up sandbox directory that has just been created.
+        # Clean up empty sandbox directory that has just been created.
         shutil.rmtree(SANDBOX_DIR)
```

Comment:
So this is the last `shutil.rmtree` call left. I cannot wrap my head around mocking a `mkdir` call.

To minimize invasiveness, I made this dir located in parent folder (`../oppia`) and also it is just an empty sandbox folder, nothing was in it or put in it at any point!